cmsplugin_filer_link/forms.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django import forms
+
+from djangocms_attributes_field.widgets import AttributesWidget
+
+from .models import FilerLinkPlugin
+
+
+class FilerLinkForm(forms.ModelForm):
+
+    class Meta:
+        model = FilerLinkPlugin
+        exclude = []
+
+    def __init__(self, *args, **kwargs):
+        super(FilerLinkForm, self).__init__(*args, **kwargs)
+        self.fields['link_attributes'].widget = AttributesWidget(val_attrs={'style': 'width: 500px!important'})
```

Comment:
Can't, because there is a "X" widget at the end of it that would wrap. I've selected this width because it works with the minimum window width as set by admin-style.